internal/db/resources_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Canonical.
```

Comment:
We call the db files resources, but the jimm pkg resource, can we make it consistent?